client/hosting/performance/site-performance.tsx

```diff
@@ -310,8 +303,8 @@ const SitePerformanceContent = () => {
 					value={ activeTab }
 				/>
 			</div>
-			{ isInitialLoading && isSitePublic ? (
-				<PerformanceReportLoading isLoadingPages isSavedReport={ false } pageTitle="" />
+			{ isLoadingPages && isSitePublic ? (
+				<PerformanceReportLoading isLoadingPages />
```

Comment:
isSavedReport and pageTitle are optional [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/performance-profiler/pages/loading-screen/progress.tsx#L122) and [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/performance-profiler/pages/loading-screen/progress.tsx#L119) so we can make them optional outside and avoid forcing code to provide `isSavedReport={ false } pageTitle=""`. Initially it was confusing for me, and now, IMO, it's clear that this component just renders info according to `isLoadingPages` property.